### PR TITLE
SB EObject ordering & Sequence messages icons

### PR DESCRIPTION
**Pull request by arnauddieumegard**

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/IImageKeys.java [hunk 1]
```diff
@@ -39,4 +39,13 @@
    */
   public final static String IMG_PRIMITIVE_VARIABLES = "variable_tab.gif";
 
+  /**
+   * Image id used in semantic browser to action "lexicographic sort" when sorting is enabled.<br>
+   */
+  public final static String IMG_LEXICOGRAPHIC_SORT_ENABLED = "full/clcl16/capella_sort.gif"; //$NON-NLS-1$
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/IImageKeys.java:45]: can you also update the licence.txt file in the project with a reference towards the original core.ui.resource image

File: core/plugins/org.polarsys.capella.core.data.gen.edit.decorators/src/org/polarsys/capella/core/data/interaction/provider/SequenceMessageItemProviderDecorator.java [hunk 2]
```diff
@@ -29,8 +33,37 @@ public SequenceMessageItemProviderDecorator(AdapterFactory adapterFactory) {
...
+            imagePath = "full/obj16/SequenceMessageReply"; //$NON-NLS-1$
+            break;
+        case CREATE:
+            imagePath = "full/obj16/SequenceMessageCreate"; //$NON-NLS-1$
+            break;
+        case DELETE:
+            imagePath = "full/obj16/SequenceMessageDelete"; //$NON-NLS-1$
+            break;
+        case TIMER:
+            if (((SequenceMessage) object).getReceivingEnd().getEvent() instanceof ArmTimerEvent) {
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.data.gen.edit.decorators/src/org/polarsys/capella/core/data/interaction/provider/SequenceMessageItemProviderDecorator.java:57]: can you do some null checks on getReceivingEnd as it occured many times that messages was badly formed.

**Review by pdulth — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/view/SemanticBrowserViewerComparator.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
+package org.polarsys.capella.core.ui.semantic.browser.view;
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/view/SemanticBrowserViewerComparator.java:1]: can you ad copyright

File: core/plugins/org.polarsys.capella.core.data.gen.edit.decorators/src/org/polarsys/capella/core/data/interaction/provider/SequenceMessageItemProviderDecorator.java (hunk 2 above)
Comment (arnauddieumegard) [reply core/plugins/org.polarsys.capella.core.data.gen.edit.decorators/src/org/polarsys/capella/core/data/interaction/provider/SequenceMessageItemProviderDecorator.java]: Done

File: core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/view/SemanticBrowserViewerComparator.java (hunk 3 above)
Comment (arnauddieumegard) [reply core/plugins/org.polarsys.capella.core.ui.semantic.browser/src/org/polarsys/capella/core/ui/semantic/browser/view/SemanticBrowserViewerComparator.java:13]: Done

**Review by pdulth — APPROVED** [review]

**Pull request merged**